kraken-build/src/kraken/std/buffrs/manifest.py

```diff
@@ -27,10 +27,11 @@ def read(cls, path: Path) -> BuffrsManifest:
 
     @classmethod
     def of(cls, path: Path, data: dict[str, Any]) -> BuffrsManifest:
+        package = data.get("package")
```

Comment:
```suggestion
        package = data.get("package", None)
```
Might this be needed in case "package" isn't a key?

Reply:
I think we are good: from [this](https://docs.python.org/3/library/stdtypes.html#dict):

> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).

This would have happened if we had used `data["package"]` which can raise `KeyError`